package.json

```diff
@@ -35,6 +35,7 @@
     "@tanstack/react-query": "^5.40.1",
     "@tanstack/react-table": "^8.10.7",
     "@wagmi/core": "^2.10.5",
+    "bun": "^1.1.13",
```

Comment:
is `bun` a required dependency?

if no, please remove to keep `build times` fast ⚡